src/lib/google.ts

```diff
@@ -1,13 +1,3 @@
-/**
```

Comment:
Only the API-KEY was supposed to be removed. Comments should be there for any dev, and also add it to the ReadMe.md for users.

Reply:
@sirajalam049 resolve